fact_extractor/unpacker/unpackBase.py

```diff
@@ -77,7 +91,13 @@ def _extract_files_from_file_using_specific_unpacker(self, file_path: str, tmp_d
         self.change_owner_back_to_me(directory=tmp_dir)
         meta_data['analysis_date'] = time()
 
-        return get_files_in_dir(tmp_dir), meta_data
+        out = get_files_in_dir(tmp_dir)
+
+        if self.exclude:
+            # Remove paths that should be ignored
+            out = filter(lambda x: not self._should_ignore(x), out)
```

Comment:
We try to avoid filter, map and reduce, because these functions are considered "unpythonic". Could you replace the expression with a list comprehension, e.g. `out = [f for f in out if not self._should_ignore(f)]`?